ospd/scan.py

```diff
@@ -63,6 +64,9 @@ def __init__(self) -> None:
         )  # type: Optional[multiprocessing.managers.SyncManager]
         self.scans_table = dict()  # type: Dict
 
+    def init_data_manager(self):
```

Comment:
Personally I wont call it init_data_manager because the data manager should be an internal detail of the class. If we are using a data manager or if we are storing the data in a dict, db, redis, memory, json, ... should not matter for the outside. For the outside using code it must only be obvious that a `init` function has to be called after the instance has been created.